.github/workflows/validate-helm-chart.yml

```diff
@@ -0,0 +1,44 @@
+name: Check Helm Generation
+
+on: 
+  push:
+    branches:
+      - 'main'
+      - '[0-9]+.[1-9][0-9]*.x'
+  pull_request:
+    branches:
+      - 'main'
+      - '[0-9]+.[1-9][0-9]*.x'
+jobs:
+  helm-generation-check:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    
+    - name: Install Helm
```

Comment:
We'll have to provide a GH action token here with no scopes.

Reply:
Since we should not use a latest version here, we should also not need a token. Please use the helm version that's used in other pipelines.

Reply:
Hey @mowies! I can't seem to find any mention of the helm version in other pipelines. Can you help me find it? I just found a mention of the helm version to be `v3.11.2` in the [Makefile](https://github.com/keptn/lifecycle-toolkit/blob/main/Makefile#L8) and it is the latest version.

Reply:
if there's no other use of the setup helm action , you can also remove it in your PR. the github runners have the Helm CLI installed anyways.